Documentation/tutorial/validate your markdown files.md

```diff
@@ -0,0 +1,128 @@
+# Validate your markdown files
+
+In GFM, we can write any document with right syntax. For example, we can write html tag `<h1>title</h1>` instead of markdown syntax `#title`. But for some purpose, some behaviors are unwanted, so we add markdown validators for each markdown document.
+
+In this document, you'll learn how to define markdown validation rules, which will help you to validate markdown documents in an efficient way.
+
+*This is part of DFM, if you switch markdown engine to other engine, validation might not work.*
+
+## How to config markdown validation rules
+
+Create a *JSON* file with name `md.style` in the same folder of `docfx.json`, and with following content:
+```json
+{
+   "rules": [],
+   "tagRules": []
+}
+```
+
+## Build-in html tag validation rules
+
+For most cases, we might need some html tag rules, and we have a build-in tag rules.
+
+For example, create `md.style` with following content:
+```json
+{
+   "tagRules": [
+      {
+         "tagName": [ "H1", "H2" ],
+         "behavior": "Warning",
+         "messageFormatter": "Some message.",
+         "customValidatorContractName": null,
+         "openingTagOnly": true
+      }
+   ]
+}
+```
```

Comment:
So we're only validating HTML tags, the content between open/close tags are not in the scope of validation, correct?